ddtrace/_trace/trace_handlers.py

```diff
@@ -675,6 +676,40 @@ def _set_span_pointer(span: "Span", span_pointer_description: _SpanPointerDescri
     )
 
 
+def _set_azure_function_tags(span, azure_functions_config, function_name, trigger):
+    span.set_tag_str(COMPONENT, azure_functions_config.integration_name)
+    span.set_tag_str(SPAN_KIND, SpanKind.SERVER)
```

Comment:
Should this span be a `server` type? Asking because I see the PR also registers a new span type called "serverless" for dd-trace-py.

Reply:
That's correct, the spans have **kind** `server` in the sense that they come from a server integration, like a flask app, as opposed to a client integration.

The **type** `serverless` adds the `fx` icon for spans in the UI like we have with spans from AWS Lambda.

<img width="1052" alt="Screenshot 2024-12-10 at 4 38 26 PM" src="https://github.com/user-attachments/assets/585789bf-45ba-4b97-86d1-0bcee25554ef" />

Reply:
Sounds good, resolving this comment 👍